src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -181,6 +182,56 @@ public static void CopyToShorterArray()
             Assert.Equal<int>(expected, dst);  // CopyTo() checks for sufficient space before doing any copying.
         }
 
+        private static void CopyTo_LargeSize_TestHelper(long BufferSize)
+        {
+            int GuidCount = (int)(BufferSize / Unsafe.SizeOf<Guid>());
+
+            unsafe
+            {
+                bool allocatedFirst = AllocationHelper.TryAllocNative((IntPtr)BufferSize, out IntPtr memBlockFirst);
+                bool allocatedSecond = AllocationHelper.TryAllocNative((IntPtr)BufferSize, out IntPtr memBlockSecond);
```

Comment:
I have submitted changes using online editor as my sandbox is being updated for another issue... Please review.